settings.py

```diff
@@ -52,7 +52,7 @@
         'ATOMIC_REQUESTS': True,
         'TEST': {
             'CHARSET': 'utf8',
-            'COLLATION': 'utf8_general_ci',
+            'COLLATION': 'utf8_distinct_ci',
```

Comment:
Why is this needed? The test runner will run the migrations and therefore apply the collation to the tables needing it.